lib/liquid/standardfilters.rb

```diff
@@ -125,7 +125,7 @@ def sort(input, property = nil)
       elsif ary.empty? # The next two cases assume a non-empty array.
         []
       elsif ary.first.respond_to?(:[]) && !ary.first[property].nil?
-        ary.sort { |a, b| a[property] <=> b[property] }
+        ary.sort { |a, b| a[property] && b[property] ? a[property] <=> b[property] : a[property] ? -1 : 1 }
```

Comment:
I can break it up across a few lines instead of using compound ternary statements. Basically, its just saying, if we have values for both properties, then use the combined comparison operator, otherwise, if the first element has the value, its considered larger.